modules/local/patchextraction.nf

```diff
@@ -11,7 +12,11 @@ process PATCHEXTRACTION{
     script:
     """
     histopreprocess.py \\
-        --inputimage ${image} 
+        --inputimage ${image} \\
+        --inputCannyValues ${params.CannyValues[0]} ${params.CannyValues[1]}\\
+        --inputCleaningThreshold ${params.CleaningThreshold} \\
+        --inputPatchPixelSize ${params.PixelSize} \\
```

Comment:
I suggest changing the name of these, they are too similar.
use for example tile_size, canny_values, threshold_cutoff
and everything in lowercase, "_" increases readability